frontend/src/routes/project/[project_code]/+page.svelte

```diff
@@ -8,93 +8,141 @@
 	import t from '$lib/i18n';
 	import type { PageData } from './$types';
 	import DeleteModal from '$lib/components/modals/DeleteModal.svelte';
-	import { _deleteProjectUser, type ProjectUser } from './+page';
+	import {
+		_changeProjectDescription,
+		_changeProjectName,
+		_deleteProjectUser,
+		type ProjectUser,
+	} from './+page';
+	import EditableText from '$lib/components/EditableText.svelte';
+	import { Badge, BadgeList, TaggedBadge } from '$lib/components/Badges';
+	import { z } from 'zod';
+	import { user } from '$lib/user';
 
 	export let data: PageData;
 
 	$: project = data.project;
+	$: _project = project as NonNullable<typeof project>;
 
 	let deleteUserModal: DeleteModal;
-	let userToDelete: ProjectUser|undefined;
+	let userToDelete: ProjectUser | undefined;
 	async function deleteProjectUser(projectUser: ProjectUser) {
 		userToDelete = projectUser;
 		await deleteUserModal.prompt(async () => {
 			await _deleteProjectUser(project!.id, projectUser.User.id);
 		});
 	}
-</script>
 
-{#if project}
-	<div>
-		<p>
-			<span class="text-2xl">{project.name}</span>
-		</p>
+	const canManage = $user?.role == 'admin' || $user?.projects.find(project => project.code == project.code)?.role == 'Manager';
 
-		<div>
-			<span>{project.code}</span>
+	const projectNameValidation = z.string().min(1, $t('project_page.project_name_empty_error'));
+</script>
 
-			<div class="badge badge-lg"><FormatProjectType type={project.type} /></div>
-			<div class="badge badge-lg"><FormatRetentionPolicy policy={project.retentionPolicy} /></div>
+<div class="space-y-4">
+	{#if project}
+		<div class="text-3xl flex items-center gap-3 flex-wrap">
+			<span>Project:</span>
+			<EditableText
+				disabled={!canManage}
+				value={project.name}
+				validation={projectNameValidation}
+				saveHandler={(newValue) => _changeProjectName({ projectId: _project.id, name: newValue })}
+			/>
+			<BadgeList>
+				<Badge><FormatProjectType type={project.type} /></Badge>
+				<Badge><FormatRetentionPolicy policy={project.retentionPolicy} /></Badge>
+			</BadgeList>
+		</div>
 
-			<span>{$t('project_page.last_commit')} <FormatDate date={project.lastCommit} /></span>
+		<div class="divider" />
+
+		<p class="text-2xl mb-4">{$t('project_page.summary')}</p>
+
+		<div class="space-y-2">
+			<span class="text-lg"
+				>{$t('project_page.project_code')}:
+				<span class="text-primary-content">{project.code}</span></span
+			>
+			<div class="text-lg">
+				{$t('project_page.last_commit')}:
+				<span class="text-primary-content"><FormatDate date={project.lastCommit} /></span>
+			</div>
+			<div class="text-lg">{$t('project_page.description')}:</div>
+			<span>
+				<EditableText
+					value={project.description}
+					disabled={!canManage}
+					saveHandler={(newValue) =>
+						_changeProjectDescription({ projectId: _project.id, description: newValue })}
+					placeholder={$t('project_page.add_description')}
+					multiline={true}
```

Comment:
for boolean attributes that are const like this you can just declare the attribute without a value and that'll be considered true. eg: `multiline` instead of `multiline={true}`